Tools/autotest/pysim/util.py

```diff
@@ -19,7 +19,13 @@
 
 import pexpect
 
-from pymavlink.rotmat import Vector3, Matrix3
+pysim_dir = os.path.dirname(os.path.realpath(__file__))
+root_dir = os.path.realpath(os.path.join(pysim_dir, '../../..'))
+try:
+    from pymavlink.rotmat import Vector3, Matrix3
+except ImportError:
```

Comment:
i'm surprised this works. how will mavproxy.py find pymavlink?

Reply:
> i'm surprised this works. how will mavproxy.py find pymavlink?

I image this is for running with the `--no-mavproxy` command-line option

Reply:
Yes this intended to run without mavproxy

Reply:
For notice there is already the same thing into sim_vehicle.py , som dependencies have changed some we need the exception here too now